src/ghe_api.py

```diff
@@ -85,23 +67,8 @@ def get_latest_commit(netid, access_token, course):
 		return latest_commit
 
 	commits = response.json()
-	latest_raw_commit = None
-	if len(commits) == 1:
-		# student has only one commit before now, so we'll use that
-		latest_raw_commit = commits[0]
-	for commit in commits:
-		# the student has multiple commits, make sure
-		# we don't grade one of the deploy commits
-		if commit["sha"] not in release_commits_sha:
-			latest_raw_commit = commit
-			break
-
-	if latest_raw_commit is None:
-		latest_commit["commit"]["message"] = (
-			"No commits found."
-		)
-	else:
-		latest_commit["message"] = latest_raw_commit["commit"]["message"]
-		latest_commit["sha"] = latest_raw_commit["sha"]
-		latest_commit["url"] = latest_raw_commit["html_url"]
+	latest_raw_commit = commits[0]
```

Comment:
Let's make sure `commits` has length greater than 0. Otherwise this would throw an exception. Put some thing like this.
```
if len(commits) == 0:	
	latest_commit["message"] = (	
		"No commits found."	
	)
else:
	# the three lines below
```

Reply:
Good point!